File: VB/dxSample/Custom Layout Control/CustomLayoutSkinPaintStyle.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraLayout.Registrator

Namespace dxSample
    Public Class CustomLayoutSkinPaintStyle
        Inherits LayoutSkinPaintStyle

        Public Sub New(ByVal owner As DevExpress.LookAndFeel.ISupportLookAndFeel)
            MyBase.New(owner)
        End Sub
        Public Overrides Function GetControlItemSkinPainter() As DevExpress.XtraLayout.Painting.LayoutControlItemSkinPainter
            If layoutControlItemSkinPainter Is Nothing Then
                layoutControlItemSkinPainter = New CustomLayoutControlItemSkinPainter()
            End If
            Return layoutControlItemSkinPainter
        End Function
    End Class
End Namespace

File: CS/dxSample/Custom Layout Control/CustomLayoutSkinPaintStyle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraLayout.Registrator;

namespace dxSample {
    public class CustomLayoutSkinPaintStyle : LayoutSkinPaintStyle {
        public CustomLayoutSkinPaintStyle(DevExpress.LookAndFeel.ISupportLookAndFeel owner)
            : base(owner) {
        }
        public override DevExpress.XtraLayout.Painting.LayoutControlItemSkinPainter GetControlItemSkinPainter() {
            if (layoutControlItemSkinPainter == null) {
                layoutControlItemSkinPainter = new CustomLayoutControlItemSkinPainter();
            }
            return layoutControlItemSkinPainter;
        }
    }
}
